### ISSUE TITLE
[Request] Please separate material bottom bar into plugin

### ISSUE THREAD

**Issue by NathanaelA** [NativeScript/NativeScript#8079]
**Is your feature request related to a problem? Please describe.**
The new material design bottom bar component; it requires a cocoapod for the app to build.   This creates new build issues.
- Cocoa pods have to be updated after every NativeScript update.
- XCode has to download and build this component
- XCode has to create a workspace
- App has additional cocoapod that isn't needed wasting build time and app size...

In other words; can this be spun off into its own component like nativescript-camera was?   This single component wastes so much time now...

**Describe the solution you'd like**
Strip this component out into its own separate plugin...

**Comment by NickIliev** [comment]
@NathanaelA the team is planning to replace MDCTabBar (see [the tracking issue](https://github.com/NativeScript/NativeScript/issues/6967)) with another implementation which won't bring cocoapod and thus will resolve all of the issues above.  The devs are still investigating what would be the best substitution for the new implementation.

**Comment by hbg34** [comment]
We started a new project with Angular, "tns create Name --ng", android has no problem however iOS stuck at these lines:
...
Installing MDFInternationalization (2.0.0)
Installing MaterialComponents (94.5.0)

We can not fix it with any of the recommended solutions.
Please advise.
